index.html

```diff
@@ -32,6 +32,7 @@
     <div class="column">
     </div>
     <div class="column">
+        <div id="example"><script>include("articles/music_album_review.md", "example")</script></div>
```

Comment:
please change the two `example` with `music_album_review` ;-)

Reply:
done !